app/models/katello/authorization/content_view_version.rb

```diff
@@ -14,6 +14,11 @@ def readable
         view_ids = ::Katello::ContentView.readable.collect { |v| v.id }
         joins(:content_view).where("#{Katello::ContentView.table_name}.id" => view_ids)
       end
+
+      def exportable
+        view_ids = ::Katello::ContentView.exportable.collect { |v| v.id }
+        joins(:content_view).where("#{Katello::ContentView.table_name}.id" => view_ids)
```

Comment:
I think with the Authorizable module you can do `joins_authorized(Katello::ContentView, :permission_name)` - you can find the export permission name in permission_creator.rb

Reply:
Updated